lsst_efd_client/efd_utils.py

```diff
@@ -73,7 +73,7 @@ def resample(df1, df2, interp_type='time'):
         That is the length of the resulting `pandas.DataFrame` is the
         sum of the lengths of the inputs.
     """
-    df = df1.append(df2, sort=False)  # Sort in this context does not sort the data
+    df = pandas.concat([df1, df2], axis=1)  # Sort in this context does not sort the data
```

Comment:
Does this do the trick?

Reply:
Yes.  That adds a column for ever entry in either dataframe with None filled out for the fields that don't exist.  Then the interpolate will fill the None values with the appropriate value given the interpolation type